docker/README.md

```diff
@@ -66,6 +75,52 @@ You can then browse to the Docker directory:
 cd scim-examples/docker/
 ```
 
+#### Docker Swarm
+
+To use Docker Swarm to deploy, you’ll want to have run `docker swarm init` or `docker swarm join` on the target node and completed that portion of the setup. Refer to Docker’s documentation for more details.
+
+Unlike Docker Compose, you won’t need to set the `OP_SESSION` variable in `scim.env`, as we’ll be using Docker Secrets to store the `scimsession` file.
+
+You’ll still need to set the environment variable `OP_LETSENCRYPT_DOMAIN` within `scim.env` to the URL you selected during [PREPARATION.md](/PREPARATION.md). Open that in your preferred text editor and change `OP_LETSENCRYPT_DOMAIN` to that domain name.
+
+<details>
+<summary>Information for Google Workspace as your Identity Provider (IdP)</summary>
+If you’re using Google Workspace as your identity provider for provisioning, you will need to set up some additional secrets to use this functionality as detailed below. Refer to our complete Google Workspace provisioning documentation for more complete information, https://support.1password.com/scim-google-workspace. 
+
+First, edit the file located at `scim-examples/beta/workspace-settings.json` and enter in the appropriate details.
+
+Next, to create the necessary secrets for Google Workspace:
+
+```bash
+# this is the path of the JSON file you edited in the paragraph above
+cat /path/to/workspace-settings.json | docker secret create workspace-settings -
+# replace <google keyfile> with the name of the file Google generated for your Google Service Account
+cat /path/to/<google keyfile>.json | docker secret create workspace-credentials -
+
+```
+</details>
```

Comment:
Do we want to promote this to an H3 heading so it's not tucked behind a disclosure triangle? This would align with my changes to the K8s readme (which doesn't mean my way was right... maybe this is the better way, but we may want to unify on one or the other!

Reply:
I like the clarity on the headings on the K8 readme, I will transpose this formatting style.